File: scripts/commands/generic/setWaypointActiveStatus.py
import sys

def execute(objManager, player, target, args):
	ghost = player.getPlayerObject()
	if ghost is None:
		return
	
	waypoint = ghost.getWaypoint(target.getObjectId())
	if waypoint is None:
		return
	
	if waypoint.isActive():
		waypoint.setActive(False)
	else:
		waypoint.setActive(True)
	
	ghost.updateWaypoint(waypoint)
	return

File: scripts/commands/generic/setWaypointActiveStatus.js
function executeCommand(galacticManager, player, target, args) {
	var ghost = player.getPlayerObject();
	var waypoint;
	
	if(ghost == null || target == null) {
		return;
	}
	
	waypoint = ghost.getWaypoint(target.getObjectId());
	
	if(waypoint == null) {
		return;
	}
	
	if(waypoint.isActive()) {
		waypoint.setActive(false);
	} else {
		waypoint.setActive(true);
	}
	
	ghost.updateWaypoint(waypoint);
}